src/lib/core/datetime/native-date-adapter.spec.ts

```diff
@@ -334,13 +333,13 @@ describe('NativeDateAdapter', () => {
 });
 
 
-describe('NativeDateAdapter with LOCALE_ID override', () => {
+describe('NativeDateAdapter with MAT_DATE_LOCALE override', () => {
```

Comment:
can we also have tests for `LOCAL_ID` override? this should still cascade to `MAT_DATE_LOCALE`